CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
+<a name="2.0.0-"></a>
+# [2.0.0] - 2024-06-13
+
+### Features
+- Minimum iOS deployment target set to iOS 14.0
+- add Mobile SDK EU1 QA configurations
```

Comment:
We want to expose this?

Reply:
Configurations are not part of the public release so I can remove the changelog entry probably 🤔